### PR TITLE
Fix coep violation report type for CORP

### PR DESCRIPTION
**Pull request by yutakahirano**
Change "CORP" to "corp", so as to be consistent with other values.

Pointed out at:
https://crrev.com/c/2076223/11/third_party/blink/web_tests/external/wpt/html/cross-origin-embedder-policy/reporting.https.html

### CODE REVIEW

**Comment by yutakahirano** [comment]
cc: @arturjanc

**Review by arturjanc — APPROVED** [review]

**Comment by mikewest** [comment]
I think this has been incorporated into other CLs? LGTM after rebase if not. :)

**Review by mikewest — APPROVED** [review]

**Comment by mikewest** [comment]
Do you need anything more from me here? I want to make sure I'm not blocking your progress on https://github.com/whatwg/html/pull/5454. :)

**Comment by yutakahirano** [comment]
Sorry, there's no external blocker. I hope I'll have more time after the branch point.

**Comment by yutakahirano** [comment]
And thank you for the review!

**Pull request merged**
